CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## master / unreleased
-
+* [CHANGE]: Compactor: Introduce a new metric to monitor blocks marked for no compact during compaction: cortex_compactor_blocks_marked_for_no_compact_total
```

Comment:
Would be nice if we can briefly explain why when would `cortex_compactor_blocks_marked_for_no_compact_total` go up (i.e why would block get marked for no compact). This will give the Cortex admin, as well as customers reading the change log more context on the background of the metrics.